native/package.json

```diff
@@ -84,7 +84,7 @@
     "react-native-elements": "^3.4.2",
     "react-native-gesture-handler": "^2.5.0",
     "react-native-highlight-words": "^1.0.1",
-    "react-native-inappbrowser-reborn": "^3.7.0",
+    "react-native-inappbrowser-reborn": "^3.6.3",
```

Comment:
Why the downgrade? Please revert

Reply:
probably due to merging by accident. reverted